polybar-scripts/updates-void/README.md

```diff
@@ -0,0 +1,15 @@
+# Script: updates-void
+See number of updates available on Void Linux.
+
+## Configuration
+If `xbps-install -Sun` returns 'ERROR: Failed to initialize libxbps: Permission denied' this module won't work.
+To fix that you'll need to add updates-void.sh to your sudoers file (see https://askubuntu.com/a/470462/825136)
```

Comment:
Please add the `sudo` hint in your README.md instead of a link. We have this already other scripts like [system-gpu-intel](https://github.com/x70b1/polybar-scripts/tree/master/polybar-scripts/system-gpu-intel).